Feature/sans

Minotaurinmaze opened the pull request:
I need to know if my Sans command is correctly set up to target players and if not tips on how to actually make it do what it supposed to do.

Minotaurinmaze committed:
Added a command that spawns a skeleton when "/sans" is typed.

Minotaurinmaze committed:
Made it so the Sans command now allows you to target someone.

SeveralCircles commented on src/com/severalcircles/colors/commands/CommandSans.java:
```diff
@@ -0,0 +1,22 @@
+    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] args) {
+        Player player = (Player) commandSender;
+        Player target = Bukkit.getPlayer(args[0]);
+        Bukkit.broadcastMessage(ChatColor.BLUE + "getttttttttttt dunked on!");
```
Perhaps it should only say get dunked on to the player who's getting sansed.

SeveralCircles commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandSans.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5543442&bid=18174487)

SeveralCircles commented on src/com/severalcircles/colors/commands/CommandSans.java:
```diff
@@ -0,0 +1,22 @@
+        Player player = (Player) commandSender;
+        Player target = Bukkit.getPlayer(args[0]);
+        Bukkit.broadcastMessage(ChatColor.BLUE + "getttttttttttt dunked on!");
+        player.getWorld().spawnEntity(player.getLocation(), EntityType.SKELETON);
```
I think you want `target.getWorld()` here instead of `player.getWorld()`

SeveralCircles commented on src/com/severalcircles/colors/commands/CommandSans.java:
```diff
@@ -0,0 +1,22 @@
+        Player target = Bukkit.getPlayer(args[0]);
+        Bukkit.broadcastMessage(ChatColor.BLUE + "getttttttttttt dunked on!");
+        player.getWorld().spawnEntity(player.getLocation(), EntityType.SKELETON);
+        return false;
```
Remember to return true

SeveralCircles reviewed (changes requested):
In addition to the changes I suggested, please also add permissions to this command to avoid some very upset server owners.

SeveralCircles committed:
Merge branch 'develop' into Feature/Sans

SeveralCircles commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandSans.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5543442&bid=18174487)

Minotaurinmaze committed:
Fixed issues with the Sans command. It now properly allows you to target someone.
Added a new permission that allows someone to use the Sans Command.
Made it so the Sans command now only sends "getttttttttttt dunked on!" to the player that is targeted.

Minotaurinmaze committed:
Merge remote-tracking branch 'origin/Feature/Sans' into Feature/Sans

# Conflicts:
#	plugin.yml

SeveralCircles commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandSans.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5543442&bid=18174487)

Minotaurinmaze commented:
It should be fixed now with https://github.com/SeveralCircles/colors/commit/2fc9cbcf1a5a6d8db82de23e18c3eb6654ef6ec8.

BeegBrainMo reviewed (approved):
The changes were made 🥵

SeveralCircles reviewed (approved):
Double check that your permissions are in the plugin.yml and you are free to merge at will.

Minotaurinmaze committed:
Fixed issue where permission was not appearing in the plugin.yml file.

SeveralCircles commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandSans.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5543442&bid=18174487)
